### PR TITLE
display selected filters

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #257 
Dispaly pills when filters for the drink type are selected
image

### CODE REVIEW

File: assets/elm/SearchDrink.elm [hunk 1]
```diff
@@ -74,6 +75,7 @@ type Msg
     = SelectABVLevel String
     | SearchDrink String
     | UpdateFilters Criteria.State
+    | UnselectFitler FilterId
```
Comment (Danwhy) [inline assets/elm/SearchDrink.elm]: I think you have a typo here:
```suggestion
    | UnselectFilter FilterId
```

File: assets/elm/SearchDrink.elm [hunk 2]
```diff
@@ -96,6 +98,13 @@ update msg model =
         UpdateFilters state ->
             ( { model | drinkFilters = state }, Cmd.none )
 
+        UnselectFitler filterId ->
```
Comment (Danwhy) [inline assets/elm/SearchDrink.elm]:
```suggestion
        UnselectFilter filterId ->
```

File: assets/elm/SearchDrink.elm [hunk 3]
```diff
@@ -186,6 +196,40 @@ view model =
...
+            case filter of
+                Just f ->
+                    getFilterId f
+
+                Nothing ->
+                    ""
+    in
+    div [ class "ma1 dib pa2 br4 bg-cs-pink white" ]
+        [ span [ class "pr1" ] [ text filterName ]
+        , span [ class "pointer pl3 b", onClick (UnselectFitler filterId) ] [ text "x" ]
```
Comment (Danwhy) [inline assets/elm/SearchDrink.elm]:
```suggestion
        , span [ class "pointer pl3 b", onClick (UnselectFilter filterId) ] [ text "x" ]
```

**Comment by SimonLab** [comment]
Thanks for the review @Danwhy the typo should be fixed now :+1:

**Pull request merged**